src/components/Hero.js

```diff
@@ -12,34 +13,31 @@ export default function Hero() {
         <Flex.Item px={5} as={Box} width={[10 / 12, 8 / 12, 8 / 12, 5 / 12]} mx="auto">
           <Relative mt={[2, 3, 5]}>
             <HeroImage width="100%" height={null} />
-            <Absolute as={Box} top={0} left={0} width="100%" height="100%" zIndex={100} style={{pointerEvents: 'none'}}>
-              <HeroOverlay width="100%" height={null} />
-            </Absolute>
           </Relative>
         </Flex.Item>
         <Flex.Item as={Box} px={5} my={[4, 3, 0]} width={[1, 1, 1, 7 / 12]}>
           <Heading color="blue.4" mb={2} fontSize={[48, 56, 84]} fontWeight="bold">
             Primer
           </Heading>
-          <Text color="blue.2" fontSize={[4, 5, 5, 7]} lineHeight={1.25}>
-            Resources, tooling, and design guidelines for building interfaces with GitHub’s design system
+          <Text as="div" color="blue.1" fontSize={[4, 5, 5, 6]} lineHeight={1.25} mb={3}>
```

Comment:
Is `div` used here to create a block level element? If so, could we change this to a `p` for semantic's sake?